### PR TITLE
Added Kamon under Monitoring

### PR DESCRIPTION
**Pull request by dpsoft**
Added Kamon under Monitoring.  The Open Source tool for monitoring applications running on the JVM:
http://kamon.io/introduction/get-started/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -348,9 +348,11 @@ A curated list of awesome Java frameworks, libraries and software.
 
 * [AppDynamics](http://www.appdynamics.com/) - Commercial performance monitor.
 * [JavaMelody](https://github.com/javamelody/javamelody) - Open-source performance monitoring and profiling.
+* [Kamon](https://www.kamon.io/) - The Open Source tool for monitoring applications running on the JVM.
 * [New Relic](http://newrelic.com/) - Commercial performance monitor.
 * [Takipi](https://www.takipi.com/) - Commercial in-production error monitoring and debugging.
 
+
```
Comment (akullpp) [inline README.md]: Why did you add the whitespace?

File: README.md [hunk 2]
```diff
@@ -348,9 +348,11 @@ A curated list of awesome Java frameworks, libraries and software.
 
 * [AppDynamics](http://www.appdynamics.com/) - Commercial performance monitor.
 * [JavaMelody](https://github.com/javamelody/javamelody) - Open-source performance monitoring and profiling.
+* [Kamon](https://www.kamon.io/) - The Open Source tool for monitoring applications running on the JVM.
```
Comment (akullpp) [inline README.md]: URL does not exist.
Definite article is wrong since it implies that it's the one and only tool.
Spelling of open-source is wrong.

**Comment by dpsoft** [comment]
@akkullpp I've close this PR and open this #211 sorry for the mess.
